contracts/acl/ACLParams.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.4.24;
```

Comment:
Could we add the MIT header here (e.g. see https://github.com/aragon/aragonOS/blob/dev/contracts/acl/IACL.sol#L1)?

Reply:
Done! ;) Is there a rule of thumb on which file an MIT header should be added?

Reply:
I need to write more documentation on this, but on all the unpinned contracts (since they're the only ones meant to be used by apps, which frees them up to use any licensing model they want).